docs/sources/datasources/loki/query-editor/index.md

```diff
@@ -107,10 +107,13 @@ log message you're interested in.
 
 ### Tail live logs
 
-Loki supports live tailing of logs in real-time in [Explore]({{< relref "../../../explore#loki-specific-features" >}}).
+Loki supports live tailing of logs in real-time in [Explore]({{< relref "../../../explore" >}}).
 
 Live tailing relies on two Websocket connections: one between the browser and Grafana server, and another between the Grafana server and Loki server.
 
+To get started select the **Live** button in the top right corner of the Explore view.
```

Comment:
```suggestion
To start tailing logs click the **Live** button in the top right corner of the Explore view.
```

Maybe?

Reply:
The key point is that you're not _selecting_ anything, only clicking the button.